src/lib.rs

```diff
@@ -7,31 +7,33 @@ extern crate quote;
 decl_derive!([Abomonation] => derive_abomonation);
 
 fn derive_abomonation(mut s: synstructure::Structure) -> quote::Tokens {
+
```

Comment:
nit: Please remove this extra line

Reply:
Will do.

Reply:
Erm, related: I'm not running `rustfmt` or anything similar, so if there are other formatting nits (e.g. some end of lines that GH observes that I don't have) just holler and I'll make them how you like.